packages/@aws-cdk/aws-batch/lib/job-definition-image-config.ts

```diff
@@ -0,0 +1,171 @@
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as ecs from '@aws-cdk/aws-ecs';
+import * as iam from '@aws-cdk/aws-iam';
+import * as cdk from '@aws-cdk/core';
+import { TaskDefinition } from './task-definition';
+
+/**
+ * Properties of a job definition container
+ */
+export interface IJobDefinitionContainer {
```

Comment:
We reserve the `I` prefix for interfaces with methods. That means this should be `JobDefinitionContainer`.

Reply:
Got it 👍